File: QTD.SPA/src/app/_DtoModels/TreeVMs/ILAWithTestVM.ts
export class ILAWithTestVM{
  id!:any;
  name!:string;
  number!:string;
  active!:boolean;
  tests:TestTreeVM[] = [];
}

export class TestTreeVM{
  id!:any;
  testTitle!:string;
  active!:boolean;
}

File: QTD2.Infrastructure/Model/TreeDataVMs/ILAWithTestVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.TreeDataVMs
{
    public class ILAWithTestVM
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Number { get; set; }

        public bool Active { get; set; }

        public List<TestTreeVM> Tests { get; set; }
    }

    public class TestTreeVM
    {
        public int Id { get; set;}

        public string TestTitle { get; set; }

        public bool Active { get; set; }
    }
}
